source/code/plugin/ApplicationInsightsUtility.rb

```diff
@@ -70,7 +72,10 @@ def initializeUtility()
           @@Tc = ApplicationInsights::TelemetryClient.new
         elsif !encodedAppInsightsKey.nil?
           decodedAppInsightsKey = Base64.decode64(encodedAppInsightsKey)
-          @@Tc = ApplicationInsights::TelemetryClient.new decodedAppInsightsKey
+          telemetrySynchronousSender = ApplicationInsights::Channel::SynchronousSender.new appInsightsEndpoint
```

Comment:
Looks good. (default behavior is sync). 
Based on the OMS-Docker feedbacj, this PR will have to change (to set the endpoint address only if its overridden)